rc4airconbotmvp.py

```diff
@@ -95,14 +82,13 @@ def start(update, context):
         global lower_credit_limit
```

Comment:
Remove the global, it is not needed because it is a local variable (do the same for the rest too)